dockerfiles/base/scripts/base/startup_03_pre_networking.sh

```diff
@@ -87,7 +87,16 @@ get_image_manifest() {
 
   # Load images from file
   BOOTSTRAP_IMAGE_LIST=$(cat ${SCRIPTS_BASE_CONTAINER_SOURCE_DIR}/images/images-bootstrap)
-  IMAGE_LIST=$(cat /version/$1/images)
+  if [ -z "${CHE_SINGLE_PORT:-}" ]; then
+    IMAGE_LIST=$(cat /version/$1/images | sed '/IMAGE_TRAEFIK/d')
+  else
+    IMAGE_LIST=$(cat /version/$1/images)
+  fi
+  if [ -z "${CHE_MULTIUSER:-}" ]; then
+     IMAGE_LIST=$(echo "${IMAGE_LIST}" | sed '/IMAGE_KEYCLOACK/d; /IMAGE_POSTGRES/d; /IMAGE_CHE_MULTI_*/d')
```

Comment:
Typo on IMAGE_KEYCLOACK. There is no 'C' in it so it should be IMAGE_KEYCLOAK

Reply:
Yeah, I had to use the name as it's defined here https://github.com/eclipse/che/blob/master/dockerfiles/cli/version/5.20.0/images#L7

@riuvshin is it ok to change the image name there? Will it break smth that relies on this name?

Reply:
yeah... :) I think you should grep whole project for `IMAGE_KEYCLOACK` and fix all occurrences this thould be enough